forest/src/cli/client.rs

```diff
@@ -24,7 +24,7 @@ pub struct Client {
     pub snapshot_height: Option<i64>,
     pub snapshot_path: Option<String>,
     /// Skips loading import CAR file and assumes it's already been loaded.
-    /// Will use the cids in the header of the file to index the chain.
+    /// Will use the `cids` in the header of the file to index the chain.
```

Comment:
```suggestion
    /// Will use the `CIDs` in the header of the file to index the chain.
```

Reply:
CID and CIDs acronyms are already in the dictionary.

Reply:
:+1: